xbmc/PasswordManager.cpp

```diff
@@ -115,12 +115,9 @@ void CPasswordManager::SaveAuthenticatedURL(const CURL &url, bool saveToProfile)
 
 bool CPasswordManager::IsURLSupported(const CURL &url)
 {
-  if ( url.IsProtocol("smb")
-    || url.IsProtocol("nfs")
-    || url.IsProtocol("sftp"))
-    return true;
-
-  return false;
+  return url.IsProtocol("smb")
```

Comment:
@pkerling do we have any sort of code styling for multiple lines like this?

Reply:
Not as part of the guidelines, I think, but our `.clang-format` uses `ContinuationIndentWith: 4`. No aligning to the first line in any case.

Reply:
@lrusak & @pkerling updated now.